packages/nc-gui/utils/filterUtils.ts

```diff
@@ -35,6 +35,26 @@ export const comparisonOpList = [
     value: 'notnull',
     ignoreVal: true,
   },
+  {
+    text: 'contains all of',
+    value: 'allof',
+    types: ['MultiSelect'],
```

Comment:
Added a field here, so we can have more uidt-specific filters in the future.

Reply:
may use `UITypes.MultiSelect`